nb/updatecenters/licenseinfo.xml

```diff
@@ -21,6 +21,7 @@
 -->
 <licenseinfo>
     <fileset>
+        <file>extras/libs.kotlin.lsp/release/kotlin-lsp/server-1.3.3.zip.external</file>
```

Comment:
Mmm... AFAIK license for this kotlin language server is the MIT license, not APL... https://github.com/fwcd/kotlin-language-server/blob/main/LICENSE.txt